src/main/java/seedu/address/logic/commands/FindCommand.java

```diff
@@ -50,8 +50,14 @@ public CommandResult execute(Model model) {
 
         model.updateFilteredPersonList(rolePredicate.or(namePredicate).or(telegramPredicate).or(isFavouritePredicate));
 
-        return new CommandResult(
-                String.format(Messages.MESSAGE_PERSONS_LISTED_OVERVIEW, model.getFilteredPersonList().size()));
+        int numberOfPeopleListed = model.getFilteredPersonList().size();
+
+        if (numberOfPeopleListed == 0 || numberOfPeopleListed > 1) {
```

Comment:
can this be simplified to >= 0

Reply:
I think here I try to say that if `numberOfPeopleListed` is 1 then the message return is person instead of people